src/status_im2/contexts/wallet/events.cljs

```diff
@@ -0,0 +1,40 @@
+(ns status-im2.contexts.wallet.events
+  (:require [utils.re-frame :as rf]
+            [taoensso.timbre :as log]
+            [utils.security.core :as security]
+            [native-module.core :as native-module]))
+
+(rf/reg-event-fx 
```

Comment:
The more common way events are formatted in re-frame is to keep the event ID in the same line as `reg-event-fx`. I recently updated zprint to take this into account so that the args after the event ID are aligned like `re-frame.core/reg-sub` (see https://github.com/status-im/status-mobile/blob/6e897f0ea6113f8242d4180f52eae6711a9c9938/.zprintrc#L34)